ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__bundle.yaml

```diff
@@ -0,0 +1,24 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+operator:
+  bundles:
+  - dockerfile_path: deploy/olm-catalog/bundle.Dockerfile
```

Comment:
Bundles can be given names:
https://github.com/openshift/ci-tools/blob/ac6374dd405750020c2825fd522e1003fff639b4/pkg/api/types.go#L1519-L1520

This will allow us to avoid the "magical" `ci-bundle0` prefix https://github.com/openshift/ci-tools/blob/ac6374dd405750020c2825fd522e1003fff639b4/pkg/api/types.go#L1588-L1589

Reply:
This is internal to the build process and the images renamed at promotion time. Since these images are only being built for promotion nobody should be interacting with these. What benefit is there to renaming again?

Reply:
These yamls are not very descriptive and require a lot of internal knowledge of ci-tools to read. One of the ways I usually  deal with this gap in knowledge is that I grep names to see where else they appear to make connections.

In this case it's hard to know that the bundle created by the dockerfile specified in `.operator.bundles[0].dockerfile_path` is published to an image named "ci-bundle0".

And also if one stumbles upon the `.promotion.additional_images.assisted-service-operator-bundle = ci-bundle0` stanza it's not apparent in any way that this image is generated by `.operator.bundles[0].dockerfile_path` 

i.e. it's hard to make the connection between `.promotion.additional_images.assisted-service-operator-bundle = ci-bundle0` and `.operator.bundles[0].dockerfile_path` 

If instead you added an `.operator.bundles[0].as` entry with the value `assisted-service-operator-bundle-image` or some other name you could come up with that is somewhat unique, then someone could grep that string to find that the `additional_images` stanza contains that exact same string so they could make the connection.

I personally had to dig through the ci-tools source code to figure out they're connected

Reply:
This really reads like you desire to submit a request for clarification on the [docs](https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/) more than anything else.